aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -47,6 +47,19 @@ module aptos_token::token {
     const EWITHDRAW_ZERO: u64 = 22;
     const ENOT_TRACKING_SUPPLY: u64 = 23;
     const ENFT_NOT_SPLITABLE: u64 = 24;
+    const ECOLLECTION_NAME_TOO_LONG: u64 = 25;
+    const ENFT_NAME_TOO_LONG: u64 = 26;
+    const EURI_TOO_LONG: u64 = 27;
+
+    //
+    // Constants
+    //
+
+    const MAX_COLLECTION_NAME_LENGTH: u64 = 32;
+    const MAX_NFT_NAME_LENGTH: u64 = 10;
+    // URI lengths: Mean: 76.97, StdDev: 37.41, 95th%: 157, 99th%: 199 (http://www.supermind.org/blog/740/average-length-of-a-url-part-2)
+    const MAX_URI_LENGTH: u64 = 200;
```

Comment:
but ipfs links aren't your average uri length lol, tho it could be shorter on average for all we know.

Reply:
I'd propose 2048 though which is the maximum length of an url https://www.sistrix.com/ask-sistrix/technical-seo/site-structure/url-length-how-long-can-a-url-be#:~:text=The%20official%20documentation%20specifies%20a,must%20not%20exceed%202%2C048%20characters

Reply:
I would use 2048 as well